src/app/space/create/codebases/codebases-item-workspaces/codebases-item-workspaces.component.less

```diff
@@ -23,12 +23,20 @@
 
 .workspace-select {
   height: 32px;
-  margin-right: 5px;
-  width: auto;
   display: initial;
 }
 
 .link-icon {
   height: 1em;
   margin-right: -36px;
 }
+
+.no-gutters {
```

Comment:
We have utility classes `margin-left-0` and `margin-right-0` which will work for this.